src/main/java/frc/robot/RobotContainer.java

```diff
@@ -425,7 +436,10 @@ private AutoRoutine flippedFiveCoral() {
 
 
     private void configureBindings() {
-        driverController.rightBumper().whileTrue(superStructure.followPositions(() -> robotMode));
+        driverController.rightBumper().whileTrue(Commands.parallel(superStructure.followPositions(() -> robotMode)));
+//                ,
+//                Commands.repeatingSequence(
+//                        Commands.deferredProxy(() -> commandFactory.driveToClosestBranch(getClosestBranch(drivetrain.getPose()))))));
```

Comment:
Your change here doesn't seem to do anything, and has some comments from when you split this out. I would remove the unneeded parallel command, as it only as one command in it, and remove the added commented code.